installer/templates/phx_ecto/data_case.ex

```diff
@@ -44,10 +44,11 @@ defmodule <%= app_module %>.DataCase do
 
   """
   def errors_on(changeset, field) do
-    for {message, opts} <- Keyword.get_values(changeset.errors, field) do
+    Ecto.Changeset.traverse_errors(changeset, fn {message, opts} ->
       Enum.reduce(opts, message, fn {key, value}, acc ->
         String.replace(acc, "%{#{key}}", to_string(value))
       end)
-    end
+    end)
+    |> Map.get(field, [])
```

Comment:
Actually, let's go with `errors_on/1` that you have proposed. This way we get rid of the pipe and it is slightly more general purpose.